src/modules/lua_check.c

```diff
@@ -972,13 +974,12 @@ noit_lua_check_resume(mtev_lua_resume_info_t *ri, int nargs) {
         mtevL(nldeb, "lua_State(%p) -> %d [check: %p]\n", ri->coro_state,
               lua_status(ri->coro_state), ci ? ci->check: NULL);
       }
-      lua_gc(mtev_lua_lmc_L(ri->lmc), LUA_GCCOLLECT, 0);
```

Comment:
Should have `mtev_lua_gc()` here ..?

Reply:
it got moved the bottom of the function.